shigoto-sagashi/job-domain-service
# Feature ss 4

**Description:**

Created the initial structure for Job
Created endpoints to save/update/find/delete the Job entity
Created table through the Flyway for Job entity

**How to test:**

Call the endpoints and validate that all is working as expected. 
Try to use the example on Postman: [link](https://www.getpostman.com/collections/67d7406787323b00ff72)

**Release plan:**

anytime
